libevmcore/EVMSchedule.h

```diff
@@ -108,6 +109,7 @@ static const EVMSchedule EIP158Schedule = []
 static const EVMSchedule MetropolisSchedule = []
 {
 	EVMSchedule schedule = EIP158Schedule;
+	schedule.blockhashGas = 800;
```

Comment:
This can stay as it is.  In this previous allcoredev meeting, it was agreed that the gas schedule changes at the Metropolis block (inclusive), not later.